pkgs/misc/emulators/pcsx2/default.nix

```
@@ -1,17 +1,19 @@
-{ alsaLib, cmake, fetchFromGitHub, gcc-unwrapped, gettext, glib, gtk3, harfbuzz
-, libaio, libpcap, libpng, libxml2, makeWrapper, perl, pkgconfig, portaudio
-, SDL2, soundtouch, stdenv, udev, wrapGAppsHook, wxGTK, zlib
+{ alsaLib, cmake, fetchFromGitHub, fmt, gcc-unwrapped, gettext, glib, gtk3
+, harfbuzz, libaio, libpcap, libpng, libpulseaudio, libsamplerate
+, libxml2, makeWrapper, perl, pkgconfig, portaudio, SDL2, soundtouch
+, stdenv, udev, wrapGAppsHook, wxGTK, zlib
```

Since you are already touching and reformatting this - can you make this one entry per line?

```suggestion
{ alsaLib
, cmake
, fetchFromGitHub
, fmt
, gcc-unwrapped
, gettext
, glib
, gtk3
, harfbuzz
, libaio
, libpcap
, libpng
, libpulseaudio
, libsamplerate
, libxml2
, makeWrapper
, perl
, pkgconfig
, portaudio
, SDL2
, soundtouch
, stdenv
, udev
, wrapGAppsHook
, wxGTK
, zlib
```

---

Done!